modules/tinymce/src/plugins/quickbars/test/ts/browser/SelectionToolbarTest.ts

```diff
@@ -1,53 +1,51 @@
-import { ApproxStructure, Assertions, Chain, GeneralSteps, Guard, Log, Pipeline, UiFinder } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
-import { TinyApis, TinyLoader, TinyUi } from '@ephox/mcagar';
+import { ApproxStructure, Assertions, UiFinder, Waiter } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyHooks, TinySelections } from '@ephox/mcagar';
 import { SugarBody } from '@ephox/sugar';
 
 import Editor from 'tinymce/core/api/Editor';
 import LinkPlugin from 'tinymce/plugins/link/Plugin';
-import QuickbarsPlugin from 'tinymce/plugins/quickbars/Plugin';
+import Plugin from 'tinymce/plugins/quickbars/Plugin';
```

Comment:
This probably shouldn't be called `Plugin` given we're importing 2 plugins here.

Reply:
Everywhere else we've run into this during the conversion we've used full names to avoid confusion, so I'd prefer we don't use `Plugin` here both for consistency and to avoid confusion.